### ISSUE TITLE
Support gradle kotlin script

### ISSUE THREAD

**Issue by knutschlesselmann4242** [dependency-check/dependency-check-sonar-plugin#163]
We use gradle kotlin script `build.gradle.kts` and the plugin cannot find our project metadata.

Our CI system logs
```
We doesn't found any Project configuration file e.g. pom.xml, gradle.build and can not link dependencies
```
It would be great if you could detect those files as well!

**Comment by Reamer** [comment]
Can you provide a small example project under [this location](https://github.com/SonarSecurityCommunity/dependency-check-sonar-plugin/tree/master/examples). I'm not familiar with `build.gradle.kts`

**Comment by kontza** [comment]
Hi,

I don't think this is purely related to a Gradle kotlin script. We get the same warning, and our setup is as follows: CI is Bamboo running builds with Maven 3.3.9, dependency checker's version is 5.2.1, dependency check sonar plugin's version is 1.1.6, and our Sonar is 6.7.7.

**Comment by Reamer** [comment]
Hi @kontza,
Did you overwrite [`sonar.sources`](https://docs.sonarqube.org/latest/analysis/analysis-parameters/)? Please make sure, that `pom.xml` is part of `sonar.sources`.

**Comment by kontza** [comment]
Hi @Reamer,

Good call on `sonar.sources`, however after checking, our setup does not touch `sonar.sources`, so it's at its default value. I would assume, that it would be OK, then.

**Comment by Reamer** [comment]
Hi @kontza,
did you see the `pom.xml` in Sonarqube web frontend (Code-Tab)?

**Comment by kontza** [comment]
Hi,

Nope. I tried defining `sonar.sources` like this:
```
sonar.sources = src/main/java,pom.xml
```
That didn't help.

Oh, our project is a large multi module setup, if that helps in figuring out what might be the reason behind this.

**Comment by Reamer** [comment]
Checkout the [multi module maven example project](https://github.com/SonarSecurityCommunity/dependency-check-sonar-plugin/tree/master/examples/multi-module-maven). In general there is no need to overwrite `sonar.sources`.
Which version are you using?
 - dependency-check
 - sonarqube
 - dependency-check-sonar-plugin

**Comment by kontza** [comment]
Here are the versions we use:
- dependency-check = 5.2.1
- sonarqube = 6.7.7
- dependency-check-sonar-plugin = 1.1.6

**Comment by Reamer** [comment]
The version combination is good. You can test your setup with the multi module maven example project. This should work out of the box.

**Comment by Reamer** [comment]
Fixes by #184
